### ISSUE TITLE
Response UI

### ISSUE THREAD

**Issue by fishcharlie** [ProxymanApp/Proxyman#271]
After using Proxyman more, it feels like the one thing that seems a bit strange is how by default the response and request information is so tiny and almost not relevant to the application. In Charles Proxy it feels like the request and response information is the main focus and takes up the majority of the window at the bottom. The list of requests themselves are not as relevant when using Charles Proxy. Whereas in Proxyman, the information for requests and responses is almost an afterthought and only takes up part of the panel on the left side.

Is there some reason behind this that I'm totally missing? It feels really backwards to have so much of the focus be on the list of requests when the request and response seems like the important part.

**Comment by shirshak55** [comment]
Yep I also think interface like this is more better than current one. Because GUI and Human computer interaction really matters :)

![image](https://user-images.githubusercontent.com/8097377/64402511-db9fdb00-d092-11e9-90b5-2440c87dede6.png)

**Comment by NghiaTranUIT** [comment]
Thank for the feedback @fishcharlie @shirshak55 

I admit that the default size and the top-down layout seems to be small.  I suppose that I would expand the size of Request / Response by haft of screen by default.

<img width="1792" alt="Screen Shot 2019-09-06 at 15 14 02" src="https://user-images.githubusercontent.com/5878421/64412177-07ce5280-d0b9-11e9-8785-a7f78310fe15.png">

Additionally, the right panel should be expanded by default.

How do you guys think?

**Comment by shirshak55** [comment]
yea right should be expanded by default. and i think 1st and 2nd column can be same I think

**Comment by NghiaTranUIT** [comment]
Hey @fishcharlie @shirshak55 

In the latest [Proxyman 1.8.0](https://github.com/ProxymanApp/Proxyman/releases/tag/1.8.0), I improved the Response layout by making it bigger and more spacious to see the content.

Additionally, we can open this Panel in Separated Window mode, so we can open Full-Screen or move to your second monitor.

![](https://user-images.githubusercontent.com/5878421/68129645-7b230200-ff4c-11e9-8334-a51e0f0a3b96.png)

and the filter bar is now easier to interact.

![](https://user-images.githubusercontent.com/5878421/68138211-c2b08a80-ff5a-11e9-8fa1-34e98238a7b5.png)

Hopefully, it would improve our productivity and fulfill your needs. Thank you guys in advance 👍
